Containerfile

```diff
@@ -1,5 +1,11 @@
 # This image contains the baseline tools to build bootable base images.
-FROM quay.io/centos/centos:stream9@sha256:30178f32547587d82188bb3ffa27f394da32d311e590f4d7cd9a66e2953c1d68
+FROM quay.io/centos/centos:stream9
```

Comment:
The quay.io/centos/centos image is not Renovate friendly. It tags new versions with the same tag. That cause the code to break if Renovate didn't update the digest on time.

Reply:
Is there an issue or a bugzilla somwhere for that? It worries me but I don't fully understand the problem.

Reply:
Nop. I already asked the centos folks if they are planning to change this practice.  It is not going to change.  We actually practice the same method when tagging our images with the `eln` and `centos9` tags.

Reply:
Am I understanding it correctly that Renovate has troubles updating the digest for images references with a tag?

So `registry.com/image@digest` works but `registry.com/image:tag@digest` does not?

Reply:
Sorry. I didn't explain myself well.  Renovate updates the digest correctly. The problem is a raise condition.
Here is an example:
1. centos:stream9 has a new build with a new digest
2. Renovate generates PR for replacing the digest
3. PR not merged yet
4. Contributor opens a PR
5. The CI for this PR is broken because the image with the previous digest is not available